AmazonChimeSDK/AmazonChimeSDK/audiovideo/data/transcript/TranscriptEntity.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  TranscriptEntity.swift
+//  AmazonChimeSDK
+//
+//  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//  SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+
+/// See [Using Amazon Chime SDK live transcription developer guide](https://docs.aws.amazon.com/chime/latest/dg/process-msgs.html) for details about transcription message types and data guidelines
```

Comment:
Nit: three `///` ?

Reply:
not sure why this is,  but it is used across all transcript (Item, result, etc) from before. E.G
https://github.com/aws/amazon-chime-sdk-ios/blob/master/AmazonChimeSDK/AmazonChimeSDK/audiovideo/data/transcript/TranscriptItem.swift

Reply:
Got it --- three slashes is the swift form of 'javadoc'. Leave it as is.